packages/server/src/create-instance/construct-style-tags.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+import type { EmotionCache } from '@emotion/utils'
+import { generateStyleTag } from './utils'
+
+const createConstructStyleTags = (
+  cache: EmotionCache,
+  nonceString: string
+) => (criticalData: {
+  html: string,
+  styles: Array<{ ids: Array<string>, css: string }>
+}) => {
+  let styleTagsString = ''
+
+  criticalData.styles.forEach(item => {
+    styleTagsString += generateStyleTag(
+      item.key || cache.key,
```

Comment:
```suggestion
      item.key,
```

I've thought that this was also used by `createRenderStylesToString` and I've left `cache.key` here as a fallback so I could investigate this later but I've now rechecked this and it's not used there. So this should be safe to be removed.